cassandane/Cassandane/Instance.pm

```diff
@@ -1745,7 +1745,7 @@ sub stop
     push @errors, $self->_check_sanitizer_logs("asan");
     push @errors, $self->_check_sanitizer_logs("ubsan");
     push @errors, $self->_check_cores();
-    push @errors, $self->_check_syslog() unless $params{no_check_syslog};
+    push @errors, $self->_check_syslog() unless $params{no_check_syslog} || $self->{no_check_syslog};
```

Comment:
per above, if the magic attribute works properly, then this isn't necessary

Reply:
Yup, fixed